src/storm-pars/analysis/OrderExtender.cpp

```diff
@@ -142,7 +142,7 @@ std::shared_ptr<Order> OrderExtender<ValueType, ConstantType>::getBottomTopOrder
             decomposition = storm::storage::StronglyConnectedComponentDecomposition<ValueType>(matrix, options);
         }
         auto statesSorted = storm::utility::graph::getTopologicalSort(matrix.transpose(), firstStates);
-        bottomTopOrder = std::shared_ptr<Order>(new Order(&topStates, &bottomStates, numberOfStates, std::move(decomposition), std::move(statesSorted)));
+        bottomTopOrder = std::shared_ptr<Order>(new Order(topStates, bottomStates, numberOfStates, std::move(decomposition), std::move(statesSorted)));
```

Comment:
Can we do
```
std::make_shared<Order>(topStates, bottomStates, numberOfStates, std::move(decomposition), std::move(statesSorted))
```
because I'm afraid of `new`